docs/source/whatsnew/1.4.0.txt

```diff
@@ -0,0 +1,1024 @@
+Release 1.4.0
+=============
```

Comment:
Looks like the headings here are a different style from previous releases. Shall we keep them consistent?

Reply:
Not sure if this is the cause, but the spacing between sections looks off:
![image](https://user-images.githubusercontent.com/2123660/87698162-19649a00-c761-11ea-9325-972e1db9df09.png)

Reply:
And looks like the index is getting confused:
![image](https://user-images.githubusercontent.com/2123660/87699784-5df13500-c763-11ea-919b-fc3c939df803.png)